src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -171,7 +171,7 @@ private bool TryConvertToJson(string json, out object obj, ref Exception exRef)
             try
             {
                 ErrorRecord error;
-                obj = JsonObject.ConvertFromJson(json, out error);
+                obj = JsonObject.ConvertFromJson(json, false, out error);
```

Comment:
Please revert the change - we have the overload.

Reply:
Yes, you're right.